virttest/utils_misc.py

```diff
@@ -3129,6 +3129,7 @@ def verify_host_dmesg(dmesg_log_file=None, trace_re=None):
     panic_re = [r"BUG:.*---\[ end trace .* \]---"]
     panic_re.append(r"----------\[ cut here.* BUG .*\[ end trace .* \]---")
     panic_re.append(r"general protection fault:.* RSP.*>")
+    panic_re.append(r"kernel: Out of memory: Kill process .*")
```

Comment:
This line is at the bottom of the message. It is not a good place for record the message in the debug logs. Please give some pattern that can cut the whole message out.